### PR TITLE
Adjust styles to a paper theme

### PR DESCRIPTION
**Pull request by jazeee**
Extract colors and add a paper theme'd background.

Playing around, we could do something like this:

image

### CODE REVIEW

File: ui/global/global.scss [hunk 1]
```diff
@@ -160,28 +167,6 @@ span.attribute-value {
   border-color: transparent;
 }
 
-button.reset-total {
```
Comment (jazeee) [inline ui/global/global.scss:163]: Moved to `buttons.scss`

File: ui/global/global.scss [hunk 2]
```diff
@@ -1,4 +1,11 @@
 /* Generic styles */
+@import '../colors.scss';
+
+.ui-dialog {
+  background-color: $paper-color;
+  background-size: cover;
+  background-image: url(https://raw.githubusercontent.com/Roll20/roll20-character-sheets/master/Pathfinder%20Second%20Edition%20by%20Roll20/images/background.png);
```
Comment (jazeee) [inline ui/global/global.scss:7]: Once merged, I'll update this URL.

**Comment by johnlamping** [comment]
This looks so much better!

**Comment by jazeee** [comment]
This is ready for review 👍🏾

**Review by johnlamping — APPROVED** [review]

**Pull request merged**